components/uniset/ppucd-test.txt

```diff
@@ -0,0 +1,2000 @@
+# Preparsed UCD generated by ICU preparseucd.py
```

Comment:
A file like this belongs in `components/uniset/test/testdata/`

Reply:
Done.